build.gradle.kts

```diff
@@ -17,4 +17,21 @@
 plugins {
     id("eclipse")
     id("idea")
+    alias(libs.plugins.nexus.publish)
+}
+
+val nexusUsername: Provider<String> = providers.gradleProperty("nexusUsername")
+val nexusPassword: Provider<String> = providers.gradleProperty("nexusPassword")
+
+nexusPublishing {
+    repositories {
+        sonatype {
+            username = nexusUsername
+            password = nexusPassword
+
+            // central portal URLs
+//            nexusUrl.set(uri("https://ossrh-staging-api.central.sonatype.com/service/local/"))
```

Comment:
With this commented out, what happens if we attempt to publish?  Will it publish to existing Sonatype portal, or just fail?

Reply:
It will publish to the deprecated sonatype urls.